### PR TITLE
fixed incorrect key qvel in myolegs_abdomen

### PR DESCRIPTION
**Pull request by P-Schumacher**

### CODE REVIEW

File: leg/myolegs_abdomen.xml [hunk 1]
```diff
@@ -38,8 +38,8 @@
...
-        <key qpos='0 0 1.0 0.707388 0 0 -0.706825 0 0 0 -0.1652 -0.0279385 -0.041886 0.00247908 0.00101098 0.0888 0.0275069 0.136817 -0.019 -0.00117055 -0.000125295 -0.0302192 0.0395202 -0.194029 -0.2326 -0.0279385 -0.041886 0.00247908 0.0010198 1.227 0.0275069 0.136817 0.1672 -0.00117055 -0.000125295 -0.0302192 0.0395202 -0.194029' qvel='0 0 0 0 -1.5 0 0 0 0 -0.576 0 0 0 0 0.175 0 0 0.988 0 0 0 0 0 4.9066 0 0 0 0 -3.597 0 0 0.633 0 0 0 0 0'/>
+        <key qpos='0 0 1.0 0.707388 0 0 -0.706825 0 0 0 -0.2326 -0.0279385 -0.041886 0.00247908 0.00101098 1.227 0.0275069 0.136817 0.1672 -0.00117055 -0.000125295 -0.0302192 0.0395202 -0.194029 -0.1652 -0.0279385 -0.041886 0.00247908 0.0010198 0.0888 0.0275069 0.136817 -0.019 -0.00117055 -0.000125295 -0.0302192 0.0395202 -0.194029' qvel='0 -1.5 0 0 0 0 4.9066 0 0 0 0 0 0 0 -3.597 0 0 0.633 0 0 0 0 0 0.175 0 0 0 0 0.175 0 0 0.988 0 0 0 0 0'/>/>
```
Comment (vikashplus) [inline leg/myolegs_abdomen.xml:41]: Note the extra />/>

**Review by vikashplus — APPROVED** [review]
Note the />/>

File: leg/myolegs_abdomen.xml (hunk 1 above)
Comment (P-Schumacher) [reply leg/myolegs_abdomen.xml:41]: Oh i might have deleted an extra line

let me fix that

**Comment by P-Schumacher** [comment]
fixed the typo, please merge

**Pull request closed without merging**